src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBranch.java

```diff
@@ -285,4 +295,42 @@ private static boolean isZeroImpedanceBranch(PiModel piModel, boolean dc, double
             return piModel.getZ() < lowImpedanceThreshold;
         }
     }
+
+    private void updateZeroImpedanceNetworks(boolean disabled, boolean dc) {
+        if (isZeroImpedance(dc)) {
+            LfZeroImpedanceNetwork zn1 = bus1.getZeroImpedanceNetwork(dc);
+            LfZeroImpedanceNetwork zn2 = bus2.getZeroImpedanceNetwork(dc);
+            if (zn1 != null && zn2 != null) {
```

Comment:
I remember now which I added this check. It is to avoid updating dc part in case of ac calculation and the invert. as LfBus.getZeroImpedanceNetwork returns null if no zero impedance infos (spanning tree) as been queried in the model.